nixos/modules/services/web-apps/nextcloud.xml

```diff
@@ -182,6 +182,17 @@
   </para>
  </section>
 
+ <section xml:id="installing-apps-php-extensions-nextcloud">
+  <title>Installing Apps and PHP extensions</title>
+
+  <para>
+   Nextcloud apps are installed statefully through the web interface.
+
+   Some apps may require extra PHP extensions to be installed.
+   This can be configured with the <link linkend="opt-services.nextcloud.phpExtraExtensions">nextcloud.phpExtraExtensions</link> setting.
```

Comment:
`<xref linkend="opt-..." />`.

Reply:
That's confusing. The rest of the document uses the format I took. I don't know what the difference is.

Reply:
The text of the link is generated automatically by `<xref />`, while it needs to be specified with `<link>…</link>`. This is used throughout the manuals.

Reply:
Fixed